### PR TITLE
Construct Push_Server_URL inside of the SDK

### PR DESCRIPTION
**Pull request by danielpassos**
See: [RHMAP-3135](https://issues.jboss.org/browse/RHMAP-3135)

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun @matzew :eyes: ?

**Comment by danielpassos** [comment]
@edewit @corinnekrych Are you doing the same in `iOS` and `Cordova`?

**Comment by corinnekrych** [comment]
@danielpassos @edewit i cloned the issue: https://issues.jboss.org/browse/RHMAP-3267
https://issues.jboss.org/browse/RHMAP-3268

**Comment by matzew** [comment]
@danielpassos what's the best way to test this ?

**Comment by danielpassos** [comment]
@matzew Hrm, since I've updated only the [getPushServerUrl](https://github.com/feedhenry/fh-android-sdk/blob/master/fh-android-sdk/src/main/java/com/feedhenry/sdk/AppProps.java#L120-L122) and [pushRegister](https://github.com/feedhenry/fh-android-sdk/blob/master/fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java#L458-L479) still using it. I think what I did in [SDK test](https://github.com/danielpassos/fh-android-sdk/blob/ups-property/fh-android-sdk-test/src/main/java/com/feedhenry/sdk/tests/api/AppPropsTest.java#L25) is enoght. 

Anway if you want to test it in a app, you can build this PR locally and use it [pushstarter-android-app](https://github.com/feedhenry-templates/pushstarter-android-app) in an existent project (in studio).

**Comment by secondsun** [comment]
lgtm

**Comment by danielpassos** [comment]
@matzew Do you still want review that? I need to merge that to release a new version ASAP.

**Comment by matzew** [comment]
lgtm

**Comment by edewit** [comment]
@danielpassos cool one and yes should do the same for all sdk

File: fh-android-sdk-test/src/main/java/com/feedhenry/sdk/tests/api/AppPropsTest.java [hunk 1]
```diff
@@ -19,5 +20,17 @@ public void testAppProps() throws Exception {
...
         assertNull(appProps.getConnectionTag());
         assertTrue(appProps.isLocalDevelopment());
+
+        assertNotNull(appProps.getPushServerUrl());
+        assertEquals("http://localhost:9000/api/v2/ag-push", appProps.getPushServerUrl());
+
+        assertNotNull(appProps.getPushSenderId());
+        assertEquals("MY_SENDER_ID", appProps.getPushSenderId());
+
+        assertNotNull(appProps.getPushSenderId());
```
Comment (vibe13) [inline fh-android-sdk-test/src/main/java/com/feedhenry/sdk/tests/api/AppPropsTest.java]: @danielpassos getPushVariant()?

File: fh-android-sdk-test/src/main/java/com/feedhenry/sdk/tests/api/AppPropsTest.java [hunk 2]
```diff
@@ -19,5 +20,17 @@ public void testAppProps() throws Exception {
...
+        assertEquals("123456789", appProps.getPushVariant());
+
+        assertNotNull(appProps.getPushSenderId());
```
Comment (vibe13) [inline fh-android-sdk-test/src/main/java/com/feedhenry/sdk/tests/api/AppPropsTest.java]: @danielpassos getPushSecret()?

**Comment by danielpassos** [comment]
@vibe13 Thanks. Fixed here => https://github.com/feedhenry/fh-android-sdk/commit/c683a43ffe60096e031dcab905010f3b63b3d369

**Pull request merged**